python-clusters/create-gke-cluster/cluster.py

```diff
@@ -61,7 +61,7 @@ def start(self):
                 node_pool_builder.with_auto_scaling(node_pool.get('numNodesAutoscaling', False), node_pool.get('minNumNodes', 2), node_pool.get('maxNumNodes', 5))
                 node_pool_builder.with_gpu(node_pool.get('withGpu', False), node_pool.get('gpuType', None), node_pool.get('gpuCount', 1))
                 node_pool_builder.with_spot_vms(node_pool.get('useSpotVms', False))
-                node_pool_builder.with_nodepool_labels(node_pool.get('nodepoolLabels', {}))
+                node_pool_builder.with_nodepool_labels(node_pool.get("nodepoolLabels", {}))
```

Comment:
should we wait for a Python style PR to change them all?

Reply:
I think that makes sense. Reverting that now.